src/vehicle.cpp

```diff
@@ -1052,10 +1065,19 @@ void vehicle::use_controls()
         menu.addentry( toggle_camera, true, 'M', camera_on ?
                        _("Turn off camera system") : _("Turn on camera system") );
     }
+    if( has_plow ){
+        menu.addentry( toggle_plow, true, 'p', _("Toggle Plow"));
```

Comment:
lowercase `p` is already used for the option "Trigger alarm". If you don't find a key, you can always use `MENU_AUTOASSIGN` and let the menu assign a unique key on its own.